frontend/src/v5/ui/components/shared/userMenu/editProfileModal/editProfilePasswordTab/editProfilePasswordTab.component.tsx

```diff
@@ -80,10 +81,7 @@ export const EditProfilePasswordTab = ({
 
 	const onSubmitError = (apiError) => {
 		if (isNetworkError(apiError)) {
-			setExpectedError(formatMessage({
-				id: 'editProfile.networkError',
-				defaultMessage: 'Network Error',
-			}));
+			setExpectedError(NETWORK_ERROR_MESSAGE);
```

Comment:
Is there a problem with just using the <NetworkError /> component?

Reply:
Done